haskell/private/cc_wrapper.py.tpl

```diff
@@ -80,8 +80,9 @@ import sys
 import tempfile
 
 WORKSPACE = "{:workspace:}"
-CC = "{:cc:}"
-PLATFORM = "{:platform:}"
+CC = os.environ.get("CC_WRAPPER_CC_PATH", "{:cc:}")
+PLATFORM = os.environ.get("CC_WRAPPER_PLATFORM", "{:platform:}")
+CPU = os.environ.get("CC_WRAPPER_CPU", "{:cpu:}")
 INSTALL_NAME_TOOL = "/usr/bin/install_name_tool"
 OTOOL = "/usr/bin/otool"
```

Comment:
These settings of INSTALL_NAME_TOOl and OTOOL are not going to work if we ever try to cross-compile from linux to osx. But I guess it is out of scope here.

Reply:
That is true. Yes, I'd consider that out of scope for now.